plugins/jobtype/src/azkaban/jobtype/HadoopJavaJob.java

```diff
@@ -121,8 +121,27 @@ protected String getJVMArguments() {
     classPath.add(HadoopConfigurationInjector.getPath(getJobProps(),
         getWorkingDirectory()));
 
-    List<String> typeClassPath =
-        getSysProps().getStringList("jobtype.classpath", null, ",");
+    // merging classpaths from plugin.properties
+    mergeClassPaths(classPath,
+      getJobProps().getStringList("jobtype.classpath", null, ","));
```

Comment:
Is there a constant for "jobtype.classpath" somewhere that we can use instead of duplicating this String literal? Same for the other constants below.